lib/src/lib/table/components/table/table.component.ts

```diff
@@ -134,16 +148,16 @@ export class Lab900TableComponent<T extends object = object> implements OnChange
   public selectableRows: boolean;
 
   @Input()
-  public selectableRowsEnabled: boolean;
-
-  @Input()
-  public selectedItems: T[];
-
-  @Input()
-  public multiSelect: boolean;
-
-  @Input()
-  public maxSelectableRows: number;
+  public selectableRowsOptions: {
+    checkBoxColor?: ThemePalette;
+    position?: 'right' | 'left';
+    sticky?: boolean;
+    showSelectAllCheckbox?: boolean;
+    disabled?: boolean;
+    maxSelectableRows?: number;
+    selectedItems?: T[];
+    singleSelect?: boolean;
```

Comment:
Create an exportable interface for this. This will require a major version bump.

Reply:
done